src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+/**
+ * An app that interacts with the users and help them list down tasks
+ * they need to do.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private TaskList taskList;
+    private Parser parser;
+
+    public String getResponse(String command) {
+        String output = "";
+        if (command.equals("bye")) {
```

Comment:
Perhaps you can consider using switch cases to minimize the need for if-else statements as it would improve the readability of your code?